packages/local_auth/local_auth_windows/test/local_auth_test.dart

```diff
@@ -0,0 +1,171 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/services.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:local_auth_windows/local_auth_windows.dart';
+
+void main() {
+  TestWidgetsFlutterBinding.ensureInitialized();
+
+  group('LocalAuth', () {
+    const MethodChannel channel = MethodChannel(
+      'plugins.flutter.io/local_auth_windows',
+    );
+
+    final List<MethodCall> log = <MethodCall>[];
+    late LocalAuthWindows localAuthentication;
+
+    setUp(() {
+      channel.setMockMethodCallHandler((MethodCall methodCall) {
+        log.add(methodCall);
+        switch (methodCall.method) {
+          case 'getEnrolledBiometrics':
+            return Future<List<String>>.value(<String>['weak', 'strong']);
+          default:
+            return Future<dynamic>.value(true);
+        }
+      });
+      localAuthentication = LocalAuthWindows();
+      log.clear();
+    });
+
+    test('deviceSupportsBiometrics calls platform', () async {
+      final bool result = await localAuthentication.deviceSupportsBiometrics();
+
+      expect(
+        log,
+        <Matcher>[
+          isMethodCall('deviceSupportsBiometrics', arguments: null),
+        ],
+      );
+      expect(result, true);
+    });
+
+    test('getEnrolledBiometrics calls platform', () async {
+      final List<BiometricType> result =
+          await localAuthentication.getEnrolledBiometrics();
+
+      expect(
+        log,
+        <Matcher>[
+          isMethodCall('getEnrolledBiometrics', arguments: null),
+        ],
+      );
+      expect(result, <BiometricType>[
+        BiometricType.weak,
+        BiometricType.strong,
+      ]);
+    });
+
+    test('isDeviceSupported calls platform', () async {
+      await localAuthentication.isDeviceSupported();
+      expect(
+        log,
+        <Matcher>[
+          isMethodCall('isDeviceSupported', arguments: null),
+        ],
+      );
+    });
+
+    test('stopAuthentication returns false', () async {
+      final bool result = await localAuthentication.stopAuthentication();
+      expect(result, false);
+    });
+
+    group('With device auth fail over', () {
+      test('authenticate with no args.', () async {
```

Comment:
Apparently the existing tests in the implementation packages have completely incorrect names 😢  let's not duplicate the mistake here. (I suspect they made more sense at some point on an earlier version of the API, then drifted as things changed.)

Let's eliminate the two groups, and just have one 'authenticate' group, with two tests of for now: 'authenticate with no arguments passes expected defaults' (which actually passes no arguments, unlike now) and 'authenticate passes all options' (which has all non-default values for options).

Reply:
Fixed.